rename-font

```diff
@@ -15,10 +15,12 @@ args = parser.parse_args()
 delugia=fontforge.open(args.input)              
 
 # Rename the file
-delugia.fontname=args.name
+delugia.fontname="{}-Regular".format(args.name)
 delugia.familyname=args.name
 delugia.fullname=args.name
-delugia.copyright=args.name
+delugia.appendSFNTName("English (US)", "Copyright", "Need something here")
+delugia.appendSFNTName("English (US)", "UniqueID", "{} Regular".format(name))

```

Comment:
Oops, must be ``format(args.name)`` of course.
Did the tests on an older instance of your script (from blog I suppose). Sorry.